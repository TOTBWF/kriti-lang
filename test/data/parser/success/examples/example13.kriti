{ "foo": "xx", "bar": "x\{{}}x" }