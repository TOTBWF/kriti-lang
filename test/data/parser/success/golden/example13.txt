Object
    ( fromList
        [
            ( "bar"
            , String "x{{}}x"
            )
        ,
            ( "foo"
            , String "xx"
            )
        ]
    )